### ISSUE TITLE
Recommended training_params for fine tuning of Yolo_NAS

### ISSUE THREAD

**Issue by lpkoh** [Deci-AI/super-gradients#1207]
### 💡 Your Question

Hi, I am trying to fine tune Yolo-NAS-L and Yolo-NAS-M on my own object detection dataset. I am trying to ask for recommendations here for what is the best training recipe.

Currently, I am thinking of 2 ways:
1. Follow the procedure outlined by Notebook "SuperGradients Object Detection How to Connect Custom Dataset", whereby you load in a pre-defined "training_hyperparams", then alter a few parameters in a way that makes more sense for fine tuning
2. Follow the procedure outlined by Notebook "Quantization-Aware Finetuning YoloNAS on a Custom Dataset", before the part on QAT. It specifically defines set of training parameters rather than imports them.

I was wondering, which approach do you recommend? Import or alter or directly use the one in the notebook.

Also, for approach 1, how would it work for fine tuning Yolo-NAS? The notebook is about fine tuning Yolo-X. It has the line "training_hyperparams.get('coco2017_yolox')". I understand if I use "training_hyperparams.get('roboflow_yolo_nas_m')" that I will get the training params "coco2017_yolo_nas_train_params.yaml". This seems to be the right one, but would I need to make alterations to the file and if so would it be similar to the one used in the notebook for Yolo_X?

Alternatively, should I just use the training parameters outlined in the notebook on Yolo_NAS? I notice there are some slight differences with coco2017_yolo_nas_train_params.yaml

**Comment by BloodAxe** [comment]
That's a good question. 

We provide YAML files and recipes as a way to reproduce the training results. 
Each yaml file is under git control, so you keep track of all the experiments.

However, this approach is not quite useful in colab environment, where it's not that easy to manage yaml files. Therefore for notebooks we follow another approach which involves creating

```
train_params = { ..... LOT OF STUFF HERE .... }
```

Main reason for this approach - is to demonstrate how to train with SG using a reasonable amount of code one has to write.

Check out our docs to learn more about configs and how we work with them here: https://docs.deci.ai/super-gradients/documentation/source/configuration_files.html
